apps/bfd-db-migrator/src/main/java/gov/cms/bfd/migrator/app/MigratorApp.java

```diff
@@ -0,0 +1,33 @@
+package gov.cms.bfd.migrator.app;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The main application/entry point for the Database Migration system. See {@link
+ * MigratorApp#main(String[])}.
+ */
+public final class MigratorApp {
+  private static final Logger LOGGER = LoggerFactory.getLogger(MigratorApp.class);
+
+  /**
+   * This method is called when the application is launched from the command line. Currently, this
+   * is a notional placeholder application that produces NDJSON-formatted log output.
+   *
+   * @param args (should be empty. Application <strong>will</strong> accept configuration via
+   *     environment variables)
+   */
+  public static void main(String[] args) {
+    Runnable helloWorld =
+        new Runnable() {
+          public void run() {
+            LOGGER.info("Hello, World!");
+          }
+        };
+    ScheduledExecutorService executor = Executors.newScheduledThreadPool(1);
```

Comment:
Sure! I realize that the more fully-functional version is going to basically run the migrations/validations (all the while producing some signals/log data) and exit with a meaningful status. I _thought_ it might be helpful to have some continuous log output–to that end, I created a `helloWorld` runnable that publishes "Hello, World!" every three seconds. This might be missing the mark, though–I'll reshape this to log a few lines and produce a zero exist status.

Reply:
not necessarily a bad idea but I'm not convinced that it was happening with that original implementation. we can chat offline about whether we might want something more substantial than what you changed it to which looks fine to me.